django/db/models/query.py

```diff
@@ -423,13 +423,18 @@ def bulk_create(self, objs, batch_size=None):
         # this by using RETURNING clause for the insert query. We're punting
         # on these for now because they are relatively rare cases.
         assert batch_size is None or batch_size > 0
-        if self.model._meta.parents:
-            raise ValueError("Can't bulk create an inherited model")
+        # Checking all the parents share the same concrete model with the our
+        # model detects the inheritance pattern ConcreteGrandParent ->
+        # MultiTableParent -> ProxyChild. Simply checking self.model._meta.proxy
```

Comment:
where there any test failures with the simpler solution? If not, it would be nice to add test for those cases. I saw one test in `tests/bulk_create/tests.py (test_inheritance)`